pkg/monitoring/metrics/operator-controller/metrics.go

```diff
@@ -13,8 +13,3 @@ func SetupMetrics() error {
 		operatorMetrics,
 	)
 }
-
-// ListMetrics registered prometheus metrics
-func ListMetrics() []operatormetrics.Metric {
-	return operatormetrics.ListMetrics()
-}
```

Comment:
@arnongilboa is removing all of those because they all do the same: `return operatormetrics.ListMetrics()`. 
Why did we do it instead of calling it directly?